src/components/layouts/collection-page-layout.tsx

```diff
@@ -432,8 +435,11 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
 
               <div className="pt-5 md:hidden block">
                 <Fresh freshness={freshness} />
+
+                <CourseProjectCard courseProject={courseProject} />
```

Comment:
@laurosilvacom got the CourseProject call out working on the course landing page!